projects/eslint-plugin/rules/general/lib/rules/expect-assert.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * SPDX-FileCopyrightText: © 2021 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: MIT
+ */
+
+const MESSAGE = 'Every `expect()` should assert something.';
```

Comment:
Interesting that you went with capitalization here, which prevents you to imply `message` inside `context.report`